internal/controllers/topology/cluster/reconcile_state.go

```diff
@@ -804,21 +804,31 @@ func (r *Reconciler) reconcileReferencedTemplate(ctx context.Context, in reconci
 func createErrorWithoutObjectName(err error, obj client.Object) error {
 	var statusError *apierrors.StatusError
 	if errors.As(err, &statusError) {
+		var msg string
+		if statusError.Status().Message != "" {
```

Comment:
Can you please add a comment showing an example of how the original message looks like and how it includes the name of the object that failed creation? I am assuming that if why we are overriding the message.

Reply:
That's exactly why we're doing it - but I don't want to be overly specific here. I'll add comments, but I don't think we can add a useful example here.

Reply:
okay. Sounds good. The comment block that we have on the function already give an idea on why we are doing this. I guess that is fine.

Reply:
Feel free to resolve.